uelc/main/views.py

```diff
@@ -322,23 +321,28 @@ def post_library_item_edit(self, request):
             li.update(doc=doc)
         if name:
             li.update(name=name)
-
         li[0].user.clear()
         for index in range(len(users)):
             user = User.objects.get(id=users[index])
             li[0].user.add(user)
 
     def post_gate_action(self, request):
-        # posted gate lock/unlock
         user = User.objects.get(id=request.POST.get('user_id'))
         action = request.POST.get('gate-action')
         section = Section.objects.get(id=request.POST.get('section'))
+        actual_location = UserPageVisit.objects.filter(
```

Comment:
Let's not do this check. If a facilitator really wants to unlock all gates for a user, we should let them.